docs/configuration/general/cookies.rst

```diff
@@ -0,0 +1,122 @@
+.. _configuration_general_cookies:
+
+Privacy and cookies
+===================
+
+By default, Open Forms only uses (first party) `strictly necessary cookies`_ 
+(although sometimes confusingly called "functionality cookies"). These cookies 
```

Comment:
```suggestion
(although sometimes confusingly called "functional cookies"). These cookies 
```
I think that's how it's usually named?

Reply:
I took it from here: https://gdpr.eu/cookies/

I think "functional" has more the meaining of "working" cookies (as apposed to cookies that do not work).